src/main/bvmui/dist/css/main.min.css

```diff
@@ -0,0 +1,3 @@
+/* Diennea S.r.l. - Copyright 2019, all rights reserved.*/ 
```

Comment:
Actually this is not the right license.
We should use the Apache license
Just pick the header from HerdDB